core/src/commonMain/kotlin/com/ustadmobile/core/view/ContentEntryList2View.kt

```diff
@@ -12,6 +12,10 @@ interface ContentEntryList2View: UstadListView<ContentEntry, ContentEntryWithPar
      */
     fun showDownloadDialog(args: Map<String, String>)
 
+    fun selectEntry(listOfSelectedEntries: String)
```

Comment:
Let's be careful with variable naming: this function name does not match what it actually does. The name would make you think that it is going to select an entry on the view. But it is actually going to show the folder list.